app/static/app/js/components/NewTaskPanel.jsx

```diff
@@ -5,7 +5,7 @@ import PropTypes from 'prop-types';
 import Storage from '../classes/Storage';
 import ResizeModes from '../classes/ResizeModes';
 import update from 'immutability-helper';
-import PluginsAPI from '../classes/plugins/API';
```

Comment:
Mm, in theory importing the PluginsAPI module is simply to make React "happy", the PluginsAPI object is set in the global scope (see API.js line 30), so I'm surprised that it would affect the workings of the plugin.

Reply:
One thing you might have to do after changing `plugins/Dashboard.js` is to restart the development environment, since the "watch" logic of webpack might fail to load your new hook. That would explain why removing the PluginsAPI import fixed the plugin, it probably forced a reload of the module and caused a recompile.

Reply:
I restarted the development environment at it didn't help. The problem seems to be that if this import is not commented, when the other PluginsAPI hook I added is called, it somehow generates a call to `http://localhost:8000/plugins/SystemJS`, which of course doesn't exist. 

I think it has to be with the fact that my new plugin uses this `NewTaskPanel.jsx` and that one PluginsAPI hook is called inside another

Reply:
Ah, I think I know what's going on; since NewTaskPanel is loaded with SystemJS, SystemJS then tries to load itself, but fails because it's not properly mapped.

If you add the following:

```javascript
'SystemJS': { loader: 'globals-loader', exports: 'SystemJS' }
```

Next to the other globals at https://github.com/OpenDroneMap/WebODM/blob/master/app/static/app/js/classes/plugins/API.js#L26 does it fix the import?

Reply:
It worked :smile: 

I'll push the fix with the rest of the changes I intendeed to do. Thanks so much for the help!